packages/block-library/src/media-text/edit.native.js

```diff
@@ -131,16 +131,15 @@ class MediaTextEdit extends Component {
 			attributes,
 			backgroundColor,
 			setAttributes,
-		//	isMobile,
+			isMobile,
 		} = this.props;
 		const {
-		//	isStackedOnMobile,
+			isStackedOnMobile,
 			mediaPosition,
 			mediaWidth,
 			verticalAlignment,
 		} = attributes;
-		const shouldStack = false; // We are temporarily not stacking until we fix alignment buttons
-		// const shouldStack = isStackedOnMobile && isMobile; // <<< Original line
+		const shouldStack = isStackedOnMobile && isMobile;
```

Comment:
Let's keep shouldStack=false for this PR, we are re-introducing it in this PR: https://github.com/WordPress/gutenberg/pull/17547/